client/src/styles/song.css

```diff
@@ -0,0 +1,55 @@
+.song {
+    width: 100%;
+    justify-content: space-between;
+     background: #222222;
+    border-bottom: 1px solid white;
+}
+
+.song-link {
+    text-decoration: none;
+    color: #808080;
+}
+
+.song-name-td, .header1 {
+    width: 360px;
+    margin-left: 30px;
+}
+
+.song-format-td, .header2 {
+    width: 160px;
+}
+
+.song-artist-td, .header3 {
+    width: 180px;
+}
+
+.song-duration-td, .header4 {
+    width: 140px;
+}
+
+.song-uploaded-at-td, .header5 {
+    width: 160px;
+}
+
+.delete-song-button {
+    cursor: pointer;
+    background: none;
+    font-weight: bold;
+    height: 10%;
+}
+
+.delete-song-button:hover {
+    color: rgb(245, 83, 83);
+}
+
+.song-link:hover {
+    text-decoration: underline;
+}
+
+.download-song-td, .delete-song-button {
```

Comment:
don't use `!important`, try to figure out how to make this style appear without using it